docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -148,6 +148,15 @@ A typical universe has an RF of 3 or 5. At the end of the [node removal](#remove
 
 1. Updates the Master addresses flag on all other nodes to inform them of the new Master.
 
+## Fix a lagging master process
+
+If a master process is down for more than its [WAL log retention period](../../../reference/configuration/yb-master/#log-min-seconds-to-retain) (defaults to 2 hrs) and then becomes healthy, it will be unable to catch up to its peers. In this scenario, the nodes tab will show that the master is in a healthy state but YugabyteDB Anywhere generates an "under-replicated masters" alert. To fix this situation,
```

Comment:
```suggestion
If a master process is down for more than its [WAL log retention period](../../../reference/configuration/yb-master/#log-min-seconds-to-retain) (defaults to 2 hrs) and then becomes healthy, it will be unable to catch up to its peers. In this scenario, the **Nodes** tab shows that the master is in a healthy state but YugabyteDB Anywhere generates an "under-replicated masters" alert. To fix this situation, do the following:
```